staking/programs/integrity-pool/src/state/pool.rs

```diff
@@ -551,8 +549,8 @@ mod tests {
             pool_data.events[i].epoch = (i + 1) as u64;
         }
 
-        let mut positions = staking::state::positions::PositionData::default();
-        // this position should be ignored (wrong target)
+        let mut stake_positions_account = DynamicPositionArrayAccount::default();
+        let mut positions = stake_positions_account.to_dynamic_position_array(); // this position should be ignored (wrong target)
```

Comment:
I think this comment should be in another line below.

Reply:
fixed